charts/nginx-ingress/values.yaml

```diff
@@ -501,6 +501,10 @@ rbac:
   ## Configures RBAC.
   create: true
 
+  cluster:
+    ## Create ClusterRole and ClusterRoleBinding
+    create: true
+
 prometheus:
```

Comment:
Hi @hafe
Can we have this be `clusterrole.create` instead?
I feel like `cluster.create` might be confusing.

Reply:
or `clusterrbac.create`

Reply:
Sure. 

Was thinking, maybe the crb can be left as is. It really needed. It is just the content of the cr that would need modification